tvm/dmlc-core/README.md

```diff
@@ -6,7 +6,7 @@ Distributed Machine Learning Common Codebase
 [![GitHub license](http://dmlc.github.io/img/apache2.svg)](./LICENSE)
 
 
-DMLC-Core is the backbone library to support all DMLC projects, offers the bricks to build efficient and scalable distributed machine learning libraries.
+dmlc-Core is the backbone library to support all dmlc projects, offers the bricks to build efficient and scalable distributed machine learning libraries.
```

Comment:
Why change from upper cases to lower cases?

Reply:
Didn't mean to change this, I'll change the names back